Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1060,105 +872,40 @@ private string FormatAttribute(string str)
 
             if (string.IsNullOrWhiteSpace(result))
             {
-                result = "Emby";
+                result = "Jellyfin";
             }
 
             return result;
         }
 
-        protected virtual bool SupportsDualModeSockets
-        {
-            get
-            {
-                return true;
-            }
-        }
-
-        private X509Certificate GetCertificate(CertificateInfo info)
-        {
-            var certificateLocation = info == null ? null : info.Path;
-
-            if (string.IsNullOrWhiteSpace(certificateLocation))
-            {
-                return null;
-            }
-
-            try
-            {
-                if (!FileSystemManager.FileExists(certificateLocation))
-                {
-                    return null;
-                }
-
-                // Don't use an empty string password
-                var password = string.IsNullOrWhiteSpace(info.Password) ? null : info.Password;
-
-                X509Certificate2 localCert = new X509Certificate2(certificateLocation, password);
-                //localCert.PrivateKey = PrivateKey.CreateFromFile(pvk_file).RSA;
-                if (!localCert.HasPrivateKey)
-                {
-                    Logger.LogError("No private key included in SSL cert {CertificateLocation}.", certificateLocation);
-                    return null;
-                }
-
-                return localCert;
-            }
-            catch (Exception ex)
-            {
-                Logger.LogError(ex, "Error loading cert from {CertificateLocation}", certificateLocation);
-                return null;
-            }
-        }
+        protected virtual bool SupportsDualModeSockets => true;
 
         private IImageProcessor GetImageProcessor()
         {
             return new ImageProcessor(LoggerFactory.CreateLogger("ImageProcessor"), ServerConfigurationManager.ApplicationPaths, FileSystemManager, JsonSerializer, ImageEncoder, () => LibraryManager, TimerFactory, () => MediaEncoder);
         }
 
-        protected virtual FFMpegInstallInfo GetFfmpegInstallInfo()
+        private FFMpegInstallInfo GetFfmpegInstallInfo()
```

Comment:
what's the point of calling a `protected` method from `private`?.. if one would override `GetFFMpegInfo()` then `GetFfmpegInstallInfo()` wouldn't know that, or am I missing something?

Reply:
`GetFFMpegInfo()` isn't virtual so it couldn't be overridden, in any case, I've also made `GetFFMpegInfo()` private so this isn't an issue.